shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -8,11 +8,14 @@
 #import <UIKit/UIKit.h>
 
 #import "flutter/shell/platform/darwin/common/framework/Headers/FlutterChannels.h"
+#import "flutter/shell/platform/darwin/ios/framework/Headers/FlutterViewController.h"
 #import "flutter/shell/platform/darwin/ios/framework/Source/FlutterTextInputDelegate.h"
 
 @interface FlutterTextInputPlugin : NSObject
 
 @property(nonatomic, assign) id<FlutterTextInputDelegate> textInputDelegate;
+@property(nonatomic, readonly) FlutterViewController* viewController;
```

Comment:
This is a heavy import and providing a lot of access to the text input that it does't need.  Please look into how maybe textInputDelegate can solve your need or create a new protocol.

Reply:
Upon review, I think I may actually be able to get away with just `UIViewController`, so I will try that out.

Reply:
Yeah, that seems to work, so that's easy enough 😄